cli/src/test/java/org/jboss/gm/cli/DifferentJVMTest.java

```diff
@@ -291,19 +294,20 @@ public void runWithJDK8Fails() throws Exception {
         try {
             m.run(args);
         } catch (Exception e) {
-            if (v.compareTo(GradleVersion.version("5.0")) >= 0) {
-                // XXX: Ignored for 4.10 as Exception differs between Gradle versions and System.err is checked later anyway
+            if (v.compareTo(GradleVersion.version("5.0")) >= 0 && v.compareTo(GradleVersion.version("7.2")) < 0) {
```

Comment:
More specifically, if it doesn't throw the same exception here, then the check later on also fails. I think that this code is relying or Gradle internals to be the same from release to release. It's obviously throwing an exception, or the test would fail in general, but it's not always the same code path.

Reply:
Now for 7 the actual exception is `org.gradle.internal.exceptions.LocationAwareException: Task 'FOOgenerateAlignmentMetadataBAR' not found in root project 'junit6682355779458076435'.` although whether it should be unwrapped as per my other comment is another question